docs/MANUAL-TESTING.md

```diff
@@ -0,0 +1,112 @@
+Manual Testing
+==============
+
+This document describes a high-level script of manual tests to check for. We
+should aim to replace items on this list with automated Spectron test cases.
+
+Image Selection
+---------------
+
+- [ ] Cancel image selection dialog
+- [ ] Select an unbootable image (without a partition table), and expect a
+  sensible warning
+- [ ] Attempt to select an unsupported image
+- [ ] Change image selection
+- [ ] Select a Windows image, and expect a sensible warning
+
+Drive Selection
+---------------
+
+- [ ] Open the drive selection modal
+- [ ] Remove all drives
+- [ ] Switch drive selection
+- [ ] Insert a single drive, and expect auto-selection
+- [ ] Insert more than one drive, and don't expect auto-selection
+- [ ] Insert a locked SD Card and expect a warning
+- [ ] Insert a too small drive and expect a warning
+- [ ] Put an image into a removable drive and attempt to flash the image to the
+  drive that contains it
+- [ ] Attempt to flash a compressed image into a drive that is big enough to
+  hold the compressed image, but not big enough to hold the uncompressed
+  version
+- [ ] Enable "Unsafe Mode" and attempt to select a system drive
```

Comment:
I guess we should also check that if unsafe mode is selected, and if only one system drive (and no removable drives) are present, we shouldn't expect auto-selection? (which obviously can't be tested on computers with more than one system drive)

Reply:
Added!